### PR TITLE
Push Code

### PR DESCRIPTION
**Pull request by catdemian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,73 @@
...
+
+### Build Node-Red AMI using Packer
+
+From the execution on terraform-net get the value of the following output:
+
+   "aws_subnet_output"
+
+1. Access terraform-net directory `cd ../packer/`
+
+2. Provide subnet_id  in the packer.json file with the value of "aws_subnet_output" just gathered before.  `vi vi packer.json`
```
Comment (diegows) [inline README.md:47]: Can we use environment variables to avoid modifications on files on git?

File: packer/packer.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "variables": {
+    "aws_access_key": "Your AWS access key",
```
Comment (diegows) [inline packer/packer.json]: Environment variables here are better, to avoid pushing secrets to the repo accidentally.

File: packer/packer.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "region": "sa-east-1",
+    "ssh_username": "ubuntu",
+    "base_ami": "ami-0f7c76bfc85e5b4ea",
+    "instance_type": "t2.micro",
+    "subnet_id": "write a value here from terraform-net output"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "access_key": "{{user `aws_access_key`}}",
```
Comment (diegows) [inline packer/packer.json:14]: same here

File: terraform-net/variables.tf [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  description = "Availability zones"
+  default     = "sa-east-1a"
+}
+
+variable "public_key_path" {
+  description = "Public key path"
+  default = "~/.ssh/id_rsa.pub"
+}
+
+variable "secret_key" {
```
Comment (diegows) [inline terraform-net/variables.tf]: same here

**Review by diegows — CHANGES_REQUESTED** [review]
Check inline comments.

**Comment by catdemian** [comment]
All suggestions are already implemented.

**Review by diegows — APPROVED** [review]

**Pull request merged**